circleguard/comparer.py

```diff
@@ -144,7 +146,32 @@ def _compare_two_replays(replay1, replay2):
         if (Mod.HR in replay1.mods) ^ (Mod.HR in replay2.mods):
             xy1[:, 1] = 384 - xy1[:, 1]
 
-        return Comparer._compute_data_similarity(xy1, xy2)
+        play_sequence_1 = xy1.T.copy() # Need copy, as the cross_correlate method uses an in-place modification of the matrix
+        play_sequence_2 = xy2.T.copy()
```

Comment:
any reason you can't just use `xy1` and `xy2` here?

Reply:
this was a vague comment, sorry - what I really meant was it should be named something with `xy1` and `xy2` in the name, not `play_sequence`